src/features/ExtensionCommands.ts

```diff
@@ -408,6 +423,53 @@ export class ExtensionCommandsFeature implements IFeature {
         return promise;
     }
 
+    private normalizeFilePath(filePath: string): string {
+        var platform = os.platform();
+        if (platform == "win32") {
+            // Make sure the file path is absolute
+            if (!path.win32.isAbsolute(filePath))
+            {
+                filePath = path.win32.resolve(
+                    vscode.workspace.rootPath,
+                    filePath);
+            }
+
+            // Normalize file path case for comparison for Windows
+            return filePath.toLowerCase();
+        } else {
+            // Make sure the file path is absolute
+            if (!path.isAbsolute(filePath))
+            {
+                filePath = path.resolve(
+                    vscode.workspace.rootPath,
+                    filePath);
+            }
+
+            //macOS is case-insensitive
+            if (platform == "darwin") {
+                filePath = filePath.toLowerCase();
+            }
+
+            return  filePath;
+        }
+    }
+
+    private findTextDocument(filePath: string): boolean {
+        // since Windows and macOS are case-insensitive, we need to normalize them differently
+        var canFind = vscode.workspace.textDocuments.find(doc => {
```

Comment:
tslint will complain if `doc` isn't in parens `(doc)`.  I've started work with tslint but man, there is soo much change.  And somewhere along the line my lint warning fixes broke various commands.  Anyway, we should still probably try to avoid adding new stuff for the linter to complain about.  :-)

Reply:
Fixed!